core/services/pipeline/task.eth_call.go

```diff
@@ -32,6 +31,7 @@ type ETHCallTask struct {
 	GasFeeCap           string `json:"gasFeeCap"`
 	ExtractRevertReason bool   `json:"extractRevertReason"`
 	EVMChainID          string `json:"evmChainID" mapstructure:"evmChainID"`
+	UnlimitedGas        string `json:"unlimitedGas"`
```

Comment:
I think prefixing with `gas` (and grouping with the others) might be less surprising to users.
```suggestion
	GasUnlimited        string `json:"gasUnlimited"`
```

Reply:
Done